### PR TITLE
+payload-free CA

### PR DESCRIPTION
**Pull request by packetpilot**
### Overview
Simple payload-free CA installer with options to securely remove the file after it's imported into the keychain (as `security add-trusted-cert` requires a file)

### Testing
This has been tested on 10.13 Beta (17A362a) 10.12.6 and works as expected.
The postflight passes `shellcheck` without any errors.

### CODE REVIEW

File: add_certificate_authority/Makefile [hunk 1]
```diff
@@ -0,0 +1,31 @@
+##
+#   Copyright 2009 Joe Block <jpb@ApesSeekingKnowledge.net>
```
Comment (unixorn) [inline add_certificate_authority/Makefile]: Put your own name here 😄

**Review by unixorn — CHANGES_REQUESTED** [review]
One minor change and it should be good to go

**Comment by packetpilot** [comment]
@unixorn done! 👍

**Comment by unixorn** [comment]
Thanks for the contribution!

**Pull request merged**